src/kit.ts

```diff
@@ -603,13 +603,19 @@ const MSVC_ENVIRONMENT_VARIABLES = [
   'LIBPATH',
   'NETFXSDKDir',
   'Path',
+  'Platform',
```

Comment:
Please remove 'Platform' from the list. This causes some of our internal projects to fail to build. Is it required to make your clang-cl Kit work?

Reply:
No, it doesn't. I just need the `VCTools*` parts. 
I ran a full diff purely as a this-seems-like-a-good-idea thing :)
I'll remove it and if anyone ever needs it they can add it.

Reply:
Done!

Reply:
I suggest not remove it, but Comment it out with reason about 'Platform', so that future user won't  add it back